### PR TITLE
[tests] Add tests for Event APIs

### PR DESCRIPTION
**Pull request by cuiyanx**
Signed-off-by: Cui, Yan yanx.cui@intel.com

### CODE REVIEW

**Comment by grgustaf** [comment]
@jprestwo, you should try this out. I get 10/18 pass on qemu and 11/18 pass on A101. "default number of listeners" is the extra one that works.

File: tests/test-event.js [hunk 1]
```diff
@@ -0,0 +1,176 @@
+// Copyright (c) 2016, Intel Corporation.
```
Comment (grgustaf) [inline tests/test-event.js]: Make this 2016-2017

**Comment by jprestwo** [comment]
This looks good to me. I got the same results as @grgustaf, so its seems it uncovers some bugs in events that I will have to fix.

**Comment by grgustaf** [comment]
+1 based on jprestwo being content. :)

**Pull request merged**